rwengine/src/render/TextRenderer.cpp

```diff
@@ -287,7 +344,7 @@ void TextRenderer::renderText(const TextRenderer::TextInfo& ti,
     dp.start = 0;
     dp.blendMode = BlendMode::BLEND_ALPHA;
     dp.count = gb.getCount();
-    auto ftexture = renderer->getData()->findSlotTexture("fonts", fonts[ti.font]);
+    auto ftexture = renderer->getData()->findSlotTexture("fonts", fontMetaData.textureName);
     dp.textures = {ftexture->getName()};
     dp.depthWrite = false;
 
```

Comment:
Only for the pager font:
I'd suggest to add `glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MAG_FILTER, GL_NEAREST);` here.
This would make its edges less blurry.
Right now its left edges are not blurred because of the modified texture coordinates at the top of this cpp file.